clmm/theory/func_layer.py

```diff
@@ -577,22 +577,26 @@ def compute_reduced_tangential_shear(
     z_cluster : float
         Galaxy cluster redshift
     z_source : array_like, float, one-parameter function or dict
-        Source redshift information depending on the `z_src_model` parameter. The options available are:       
-            * Background source galaxy redshift(s) (array_like, float) if `z_src_model`='discrete' (default),
-            * Background source galaxy redshift distribution function (one-parameter function) if `z_src_model`=`distribution`,
-            * Lensing efficiency parameter(s) averaged over the galaxy redshift distribution (dict) if `z_src_model`=`averaged lensing efficiency`.
-            The dictionnary should contain a 'beta_s_mean' key and a 'beta_s_square_mean' key with the appropriate corresponding values :
-                 *beta_s_mean: array_like, float
-                    Lensing efficiency averaged over the galaxy redshift distribution. 
+        Source redshift information depending on the `z_src_model` parameter:
```

Comment:
Thank you @m-aguena for editing the docstrings. For `z_source` description I think it is better to not start by the`z_src_model` value as it is a bit misleading.